modules/apps/friendly-url/friendly-url-service/src/test/java/com/liferay/friendly/url/internal/util/FriendlyURLNormalizerImplTest.java

```diff
@@ -194,6 +195,46 @@ public void testNormalizeWithEncodingUnicodeMalformed() throws Exception {
 				"\uDBFF-" + value));
 	}
 
+	@Test
+	public void testNormalizeWithPeriods() {
+		String s1 = RandomTestUtil.randomString();
+		String s2 = RandomTestUtil.randomString();
+
+		Assert.assertEquals(
+			_friendlyURLNormalizerImpl.normalize(s1 + s2),
+			_friendlyURLNormalizerImpl.normalizeWithPeriods(s1 + s2));
+
+		Assert.assertEquals(
+			_friendlyURLNormalizerImpl.normalize(s1 + StringPool.DASH + s2),
+			_friendlyURLNormalizerImpl.normalizeWithPeriods(
+				s1 + StringPool.PERIOD + s2));
+
+		Assert.assertEquals(
+			_friendlyURLNormalizerImpl.normalize(s1 + StringPool.SLASH + s2),
+			_friendlyURLNormalizerImpl.normalizeWithPeriods(
+				s1 + StringPool.SLASH + s2));
+	}
+
+	@Test
+	public void testNormalizeWithPeriodsAndSlashes() {
+		String s1 = RandomTestUtil.randomString();
```

Comment:
I have my doubts about this names :D

Reply:
Me too :D I will try to improve them! Thanks!

Reply:
I have added one more commit. Could you please review it again? 🙏 
Thanks a lot! ❤️